templates/endpoints_v2/README.md

```diff
@@ -153,9 +153,9 @@ Look at the following guides for more advanced use-cases:
 * [Learn how to leverage different configurations that can optimize the latency and throughput of your models](examples/OptimizeModels.ipynb)
 * [Learn how to fully configure your deployment including auto-scaling, optimization parameters and tensor-parallelism](examples/AdvancedModelConfigs.ipynb)
 
-## Application Examples
+## Application examples
 
-See examples of building applications with your deployed endpoint on the [Anyscale Endpoints](https://docs.endpoints.anyscale.com/category/examples) page.
+See examples of building applications with your deployed endpoint on the [Anyscale Endpoints](https://docs.anyscale.com/examples) page.
```

Comment:
This links to the AICA page.

Reply:
This page is the start of what the user is looking for (because we don't have a landing page): https://docs.anyscale.com/examples/work-with-openai